src/cs50/sql.py

```diff
@@ -32,12 +33,23 @@ def __init__(self, url, **kwargs):
             if not os.path.isfile(matches.group(1)):
                 raise RuntimeError("not a file: {}".format(matches.group(1)))
 
-        # Create engine, raising exception if back end's module not installed
-        self.engine = sqlalchemy.create_engine(url, **kwargs)
+            foreign_keys_enabled = kwargs.pop("foreign_keys_enabled", False)
```

Comment:
Missing comment atop this line?

Reply:
Where did `foreign_keys_enabled` come from, is that your choice of arg names? Or borrowed from SQLAlchemy? If former, should probably be just `foreign_keys`, for consistency with the PRAGMA directive, http://docs.sqlalchemy.org/en/latest/dialects/sqlite.html#foreign-key-support?